src/main.ts

```diff
@@ -35,39 +34,43 @@ interface Settings {
   apiKey: string;
   filter: string;
   syncAt: string;
-  frequency: number;
   customQuery: string;
   highlightOrder: string;
   articleTemplate: string;
   highlightTemplate: string;
   syncing: boolean;
   folder: string;
-  intervalId: number;
   dateFormat: string;
 }
 
 const DEFAULT_SETTINGS: Settings = {
   apiKey: "",
   filter: "HIGHLIGHTS",
   syncAt: "",
-  frequency: 60,
   customQuery: "",
-  articleTemplate: `[{{{title}}}]({{{omnivoreUrl}}})
-  site: {{#siteName}}[{{{siteName}}}]{{/siteName}}({{{originalUrl}}})
-  {{#author}}
-  author: {{{author}}}
-  {{/author}}
-  {{#labels.length}}
-  labels: {{#labels}}[[{{{name}}}]]{{/labels}}
-  {{/labels.length}}
-  date_saved: {{{dateSaved}}}`,
+  articleTemplate: `---
+{{#author}}
+author: {{{author}}}
+{{/author}}
+{{#labels.length}}
+tags:
+{{#labels}}  - {{{name}}}
```

Comment:
Oh this is nice, so its a yaml list. One issue I wonder about, is I think tags have to be a single word in Obsidian. Will see what happens with some of my two word labels.

Reply:
Good catch.

According to Obsidian docs:
```
Spaces are not allowed in tags. So, to differentiate two or more words in a tag, you can use these case styles/formats:

camelCase: [#twoWords](https://publish.obsidian.md/#twoWords)
PascalCase: [#TwoWords](https://publish.obsidian.md/#TwoWords)
snake_case: [#two_words](https://publish.obsidian.md/#two_words)
kebab-case: [#two-words](https://publish.obsidian.md/#two-words)
```

So I will use camel case